src/templates/import.js

```diff
@@ -684,6 +684,50 @@ module.exports = (router) => {
     });
   };
 
+  const tryToLoadComponent = (component, template) => {
+    const query = {
+      name: component.form,
+      deleted: {$eq: null}
+    };
+    return formio.resources.form.model.find(query).exec().then((results)=>{
+      if (results.length!==1) {
+       throw 'Incorrect project template';
+      }
+      const result = results[0];
+
+      const newItem = {
+          "title": result.title ,
+          "type": result.type,
+          "name": result.name,
+          "path": result.path,
+          "tags": result.tags,
+          "components": result.components
+      };
+
+      if (result.type==='form') {
+        template.forms[newItem.name]=newItem;
+      }
+ else {
+        template.resources[newItem.name]=newItem;
+      }
+    });
+  };
+
+  const checkTemplate = (template) => {
+    const test = Object.values(template.forms).concat(Object.values(template.resources));
+    const resultArr = [];
+    test.forEach(form=>{
+      return form.components.forEach(component=>{
```

Comment:
This will solve the problem only for the first-level components. But if I have a Panel and a nested form inside it, it won't work.